app/views/tweets/show.erb

```diff
@@ -0,0 +1,21 @@
+
+<h1>Welcome, <%=@user.username%></h1>
+
+<h3>Your Fwitter digest:</h3>
+
+<%Tweet.all.reverse.each do |tweet|%>
```

Comment:
we should leave database logic out of the view and set this to an instance variable in the TweetsController

`@tweets = Tweet.all`